src/components/shared/Promotional/Promotional.js

```diff
@@ -50,7 +47,7 @@ export const Promotional = ({
           </div>
           <div className="col-lg-6 col-md-12">
             <figure className="dp-img-promotion">
-              <img src={_('common.ui_library_image', { imageUrl: previewUrl })} alt={title} />
+              <img src={previewUrl} alt={title} />
```

Comment:
We try to use images as styled components. Like for example [here](https://github.com/FromDoppler/doppler-webapp/blob/master/src/components/Integrations/Shopify/Shopify.styles.js#L7) I suppose it cannot be applied in this code because of the parameter, can it?

Reply:
I think it can. I will try

Reply:
It has no styles but it is coherent with other images in the app.